bidirection_streaming/client/databroker_pb2.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would prefer a script that transforms `databroker.proto` into the `.py` files and copies them to both directories, instead of checking the generated files in twice.